src/scripts/main.js

```diff
@@ -1,3 +1,232 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const rows = tbody.querySelectorAll('tr');
+const employeeForm = document.createElement('form');
+
+employeeForm.classList.add('new-employee-form');
+
+const selecteHandler = (e) => {
+  if (!e.target) {
+    return;
+  }
+
+  [...tbody.children].forEach(employee => {
+    employee.classList.remove('active');
+  });
+
+  const selectTr = e.target.parentNode;
+
+  selectTr.classList.add('active');
+};
+
+tbody.addEventListener('click', selecteHandler);
+
+employeeForm.insertAdjacentHTML('afterbegin', `
+  <label for="name">
+    Name:
+    <input name="name" data-qa="name" type="text" required>
+  </label>
+  <label for="position">
+    Position:
+    <input name="position" data-qa="position" type="text" required>
+  </label>
+  <label for="office">
+    Office:
+    <select name="office" data-qa="office" id="office" required>
+      <option value="Tokyo">Tokyo</option>
+      <option value="Singapore">Singapore</option>
+      <option value="London">London</option>
+      <option value="New York">New York</option>
+      <option value="Edinburgh">Edinburgh</option>
+      <option value="San Francisco">San Francisco</option>
+    </select>
+  </label>
+  <label for="age">
+    Age:
+    <input name="age" data-qa="age" type="number" required>
+  </label>
+  <label for="salary">
+    Salary:
+    <input name="salary" data-qa="salary" type="number" required>
+  </label>
+  <button type="submit">Save to table</button>
+`);
+
+document.body.append(employeeForm);
+
+const pushNotification = (
+  positionTop,
+  positionRight,
+  title,
+  description,
+  type
+) => {
+  const notification = document.createElement('div');
+  const titleNotification = document.createElement('h2');
+  const descriptionNotification = document.createElement('p');
+
+  notification.classList.add('notification', `${type}`);
+  notification.dataset.qa = 'notification';
+  titleNotification.classList.add('title');
+  descriptionNotification.classList.add('description');
+
+  titleNotification.innerText = title;
+  descriptionNotification.innerText = description;
+
+  notification.style.top = positionTop + 'px';
+  notification.style.right = positionRight + 'px';
+
+  notification.append(titleNotification, descriptionNotification);
+  document.body.append(notification);
+
+  setTimeout(() => notification.remove(), 2000);
+};
+
+const submit = employeeForm.querySelector('button');
+const input = employeeForm.querySelectorAll('input');
+const select = employeeForm.querySelector('select');
+
+const addToForm = (e) => {
+  e.preventDefault();
+
+  const inputName = input[0].value;
```

Comment:
```suggestion
  const inputName = input[0].value;
```
это не имя инпута, а значение которе ты взял из интпута
просто name

и так дале